lib/info-extras.js

```diff
@@ -133,6 +139,10 @@ exports.getRelatedVideos = (body) => {
     watchNextJson = JSON.parse(jsonStr.watch_next_response);
     rvsParams = jsonStr.rvs.split(',').map((e) => qs.parse(e));
     secondaryResults = watchNextJson.contents.twoColumnWatchNextResults.secondaryResults.secondaryResults.results;
+
+    if (!isIterable(secondaryResults)) {
```

Comment:
is `secondaryResults` `undefined` or a different object? if it's `undefined`, I'd rather check for truthy than iterable.

is there a video where you're getting this?